package.json

```diff
@@ -136,7 +136,7 @@
     "regenerator-runtime": "0.13.9",
     "sass": "1.49.7",
     "sass-loader": "10.2.1",
-    "scratch-gui": "1.8.45",
```

Comment:
Not a requirement for this story, but this will use the editor with similar style changes.